packages/@okta/vuepress-site/guides/sign-into-your-web-app/before-you-begin/index.md

```diff
@@ -1,22 +1,19 @@
 ---
-title: Sign Users into Your Web App
+title: Before You Begin
 ---
+This guide shows you how to sign users in to your web app.
 
-This guide walks you through how to sign users into your existing web app. It is available for the following frameworks:
+This guide assumes that you:
 
-* Node (Express)
-* Java (Spring)
-* .NET (ASP.NET)
-* .NET (ASP.NET Core)
-
-## Before You Begin
-These instructions assume that you: 
-
-* Are building a web app that's rendered by a server, otherwise see [Sign Users into a Single-Page App](sign-into-spa). If you are building a mobile app, see [Sign Users into a Mobile App](sign-into-mobile-app). If you are building an API, see [Protect an API].
-* Have an Okta Developer organization. If you don't, you can get one here: <https://developer.okta.com/signup>
+* Have an Okta Developer Edition organization. (Don't have one? [Create one for free](https://developer.okta.com/signup).)
+* Know the basics of building JavaScript applications.
```

Comment:
@susanharper-okta is JavaScript correct here?

Reply:
uh nope....that one can be removed......weird - not even sure where that came from...

Reply:
Yup! Perfect! Thank you.